### PR TITLE
Terraform scripts to deploy Author to ECS

### PR DESCRIPTION
**Pull request by jonnyshaw89**
These Terraform scripts deploy Author to ECS

### CODE REVIEW

**Review by samiwel — APPROVED** [review]
Don't claim to understand this all yet, but seems to work 👍

File: author-api.tf [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+}
+
+data "template_file" "author-api" {
+  template = "${file("${path.module}/task-definitions/author-api.json")}"
+
+  vars {
+    LOG_GROUP          = "${aws_cloudwatch_log_group.author-api.name}"
+    CONTAINER_REGISTRY = "${var.docker_registry}"
+    CONTAINER_TAG      = "${var.author_api_tag}"
+    DB_CONNECTION_URI  = "postgres://author:authorPassword@${aws_db_instance.author_database.address}:${aws_db_instance.author_database.port}/author"
```
Comment (ajmaddaford) [inline author-api.tf]: The authorPassword bit probably needs looking at

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# eq-author-deploy
+
+This repository hold the code that is responsible for deploying [Author](https://github.com/ONSdigital/eq-author) [Author API](https://github.com/ONSdigital/eq-author-api) and [Publisher](https://github.com/ONSdigital/eq-publisher)
```
Comment (ajmaddaford) [inline README.md]: hold -> holds

File: rds.tf [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+resource "aws_db_instance" "author_database" {
+  allocated_storage           = 10
+  identifier                  = "${var.env}-author-rds"
+  engine                      = "postgres"
+  engine_version              = "9.4.7"
+  allow_major_version_upgrade = "false"
+  instance_class              = "db.t2.small"
+  name                        = "author"
+  username                    = "author"
+  password                    = "authorPassword"
```
Comment (ajmaddaford) [inline rds.tf]: This needs to be parameterised

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**